internal/servers/controller/handlers/hosts/host_service.go

```diff
@@ -178,11 +177,7 @@ func (s Service) createInRepo(ctx context.Context, scopeId, catalogId string, it
 	}
 	out, err := repo.CreateHost(ctx, scopeId, h)
 	if err != nil {
```

Comment:
any chance we want to test that all the errors raised by the repo are infact domain errors?

Reply:
I updated the tests in this path to do an `errors.Match`-  I could add an `assertDomainErr` check before?

Reply:
The tests are great, but what will happen if something is missed an a non-domain error is returned from the repo?

Reply:
Sorry this was more my misunderstanding of the request.  Are you suggesting I put a convert before the return, as I had it in a previous version of this refactor, and only return err directly if it’s a domain error and otherwise return as it is today?